gisellemartel/CONPASS
# US3B - Multiple Calendars Synchronization

Add functionality of enabling users to synchronize multiple Google calendars.

To test US3B:
1. Go to https://calendar.google.com/calendar/r
2. Create a new calendar (press on the plus button above other calendars)
3. In the checkbox under 'My calendars' choose the newly created calendar ONLY
Add multiple events
4. Go into expo, sign into expo and synchronize your newly created calendar and your Gmail 5. calendar
5. Now you will see the events from both calendars
